src/Uno.UI/UI/Xaml/Shapes/Path.wasm.cs

```diff
@@ -30,7 +34,62 @@ protected override SvgElement GetMainSvgElement()
 						("d", gd.Data),
 						("fill-rule", gd.FillRule == FillRule.EvenOdd ? "evenodd" : "nonzero"));
 					break;
+				case PathGeometry pg:
+					_path.SetAttribute(("d", ToStreamGeometry(pg)));
+					break;
+			}
+		}
+		/// <summary>
+		/// Transform the figures collection into a SVG Path according to :
+		/// https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+		/// </summary>
+		/// <param name="geometry"></param>
+		/// <returns></returns>
+		private string ToStreamGeometry(PathGeometry geometry)
+		{
+			List<string> pathlist = new List<string>();
+
+			foreach (PathFigure figure in geometry.Figures)
+			{
+				pathlist.Add("M " + figure.StartPoint.X + "," + figure.StartPoint.Y);
```

Comment:
@GuillaumeSE This is still not correct, you have to format all doubles one by one, for the whole function.